book/website/communication/peer-review/peer-review-code.md

```diff
@@ -0,0 +1,91 @@
+- Does it run? 
+- Is it easy to understand? 
+Or is it more complex than it should be?
+- If it is not just a script underlying a publication but part of software infrastructure: how easy is it to maintain? 
+
+[Code does not have to be perfect](https://google.github.io/eng-practices/review/reviewer/standard.html) - it has to work, be accompanied with sufficient documentation and be maintainable if this is needed.
+
+[Code review can take place asynchronous or synchronous](https://uwescience.github.io/neuroinformatics/2017/10/08/code-review.html). 
+Sometimes face-to-face communication (synchronous) is easier to address any questions that arise during the review {cite:ps}`Petre2014codereview`. 
+
+The [online sustainability evaluation](https://www.software.ac.uk/resources/online-sustainability-evaluation) provided by the Software Sustainability can help address issues that affect the sustainability of the software.
+
+## Code review in [ReproHack style](https://annakrystalli.me/n8cir-reprohacks/slides/#24)
+
+*This is more applicable when you're reviewing the code underlying a research article.*
+
+**1. Access**
+- How easy is it to access the materials? 
+Can you access all the materials?
+  - Is the data stored in a separate directory or data repository? 
+Is there a persistent identifier associated with the data/code?
+
+**2. Installation**
+- Are you able to install everything, did you run into any problems and how did you solve these?
+
+**3. Documentation**
+Does the documentation contain information on:
+- how to install necessary software and dependencies?
+- how to use materials to reproduce the paper?
+- how to cite the materials, ideally in a form that can be copy and pasted?
+Provide suggestions on how to improve the documentation of the code if needed.
+- Are the comments helpful and necessary?
+Comments should explain why some code exists, not what the code is doing. [If the code isn’t clear enough to explain itself, then the code should be made simpler](https://google.github.io/eng-practices/review/reviewer/looking-for.html#comments). 
+- Is the code following applicable style guides? (for example, [Google Style Guide](http://google.github.io/styleguide/)
+
+**4. Reproduction**
+- Were you able to fully reproduce the paper? 
+- How automated was the process of reproducing the paper?
+- How easy was it to link analysis code to the plots it generates and sections in the manuscript in which it is described and results reported
+
+If you are not able to reproduce the article: 
+- Were there missing dependencies? 
+- Was the computational environment not adequately described / captured? 
+- Were there bugs in the code? 
+- Does the code handle errors properly? 
+Where could this be improved? 
+If there are any test, check if they are correct, sensible, and useful.
+- Did code run but results (such as model outputs, tables, figures) differ to those published? By how much?
```

Comment:
```suggestion
- Did code run but results (such as model outputs, tables, figures) differ to those published? By how much? Was this to be expected, e.g. because of use of random numbers in the method?
```

Reply:
Added this, but without the e.g. because we're avoiding these abbreviations. Thanks!